nncf/torch/exporter.py

```diff
@@ -81,6 +81,7 @@ def _export_to_onnx(self, save_path: str) -> None:
 
         :param save_path: The path where the model will be saved.
         """
+        original_device = getattr(self._model, "device", None)
```

Comment:
Can we use `get_model_device` ([link](https://github.com/openvinotoolkit/nncf/blob/6b233273bcf6f8942d9570fa9ab3dbba89e7ec54/nncf/torch/utils.py#L347) instead?

Reply:
That works, thanks!